code/game/machinery/doors/airlock.dm

```diff
@@ -1512,4 +1533,35 @@ About the new airlock wires panel:
 	welded = FALSE
 	locked = FALSE
 	QDEL_NULL(brace)
-	.=..()
\ No newline at end of file
+	QDEL_NULL(reinforcement)
+	return ..()
+
+/obj/machinery/door/airlock/proc/try_reinforce(obj/item/airlock_reinforcement/R, mob/M)
+	var/msg = SPAN_WARNING("You cannot apply [R] to [src] from this position.")
+	if(get_turf(R) == get_turf(src))
+		to_chat(M, msg)
+		return
+	var/form_dir = get_dir(M, src)
+	if(form_dir == GLOB.reverse_dir[dir] || form_dir == dir)
+		if(!do_after(M, 5 SECONDS, src, TRUE) || !density || !R || !M.unEquip(R, src))
+			to_chat(M, SPAN_NOTICE("fallaste"))
+			return
+		reinforcement = R
+		reinforcement.block_dir = form_dir
+		icon = 'icons/obj/doors/station/metaldoor.dmi'
+		update_icon()
+		to_chat(M, SPAN_NOTICE("You apply [R] to [src]."))
+
+/obj/machinery/door/airlock/proc/remove_reinforce(obj/item/I, mob/M)
+	reinforcement.forceMove(get_turf(src))
+	icon = initial(icon)
+	update_icon()
+	to_chat(M, SPAN_NOTICE("You remove [reinforcement] from [src]."))
+
+/obj/item/airlock_reinforcement
```

Comment:
airlock braces already exist in baycode, this is redundant, may as well just repurpose them

Reply:
> airlock braces already exist in baycode, this is redundant, may as well just repurpose them

You are right, I realized this after I finished the code.